### ISSUE TITLE
Protocol Enhancement - Change "issuer" to be equal to the address holding the Raft Token

### ISSUE THREAD

**Issue by bordumb** [otterspace-xyz/otterspace-contracts#110]
## Overview

Terminology/Hypotheticals parameters:
- Multi-Sig Owner: any address that is considered an owner/manager of a DAO's multi-sig (`multi_sig.owner_wallet_address`
- Badge Issuer: the party that gives the badge to a recipient (`badge_issuer`)
- Badge Recipient: the party that received the badge from the issuer (`badge_recipient`)
- Confer Transaction: the transaction that gives the badge to the recipient 

According to @grp06: 
The _Badge Issuer_ currently always equal to the wallet address that signs the _Confer Transaction_ 
In code terms:
```
badge_issuer = multi_sig.owner_wallet_address
```
However, the protocol does not allow for the following:
```
badge_recipient = multi_sig.owner_wallet_address
```
In plain English, this means:
The Multi-Sig Owner (i.e. `multi_sig.owner_wallet_address`) is unable to issue themselves a badge.

## Problem

This is a huge problem for any DAO that manages its governance via a multi-sig (e.g. Gnosis Safe).

It means that the managing director of a DAO is unable to get a badge.

There may be some workaround, such as temporarily swapping the raft token to someone else, then swapping back. But I'd argue that this UX is not a good workaround. A protocol is possibly a better long-term solution (see below).

## Proposal

I'd propose that the protocol redefines "Badge Issuer" to be equal to the Multi-Sig Address holding the Raft Token.
```
badge_issuer = multi_sig.multi_sig_address
```

**Comment by grp06** [comment]
fixed with multi-admin
